charts/4allportal/templates/webdav/configmap.yaml

```diff
@@ -124,4 +127,73 @@ data:
     BrowserMatch " Konqueror/4" redirect-carefully
     BrowserMatch "^gvfs" redirect-carefully
     BrowserMatch "WebDAVLib" redirect-carefully
-{{- end }}
+  log-handler.sh: |
+    #!/bin/ash
+    MOUNT_POINT="{{ $.Values.webdav.events.mountPath }}"
+    MOUNT_POINT_LEN=$(echo "$MOUNT_POINT" | awk '{print length($1) + 1;}')
+
+    ENDPOINT="http://{{ include "common.names.fullname" . }}-backend/api/extensions/dam/fs-event"
+    while IFS= read -r line; do
+      status=$(echo "$line" | cut -d';' -f1)
+      type=$(echo "$line" | cut -d';' -f2)
+      path=$(echo "$line" | cut -d';' -f3)
+      dest=$(echo "$line" | cut -d';' -f4 | awk '{
+        i = index($1,"/webdav/");
+        out = substr($1, i);
+        for (i = 0x20; i < 0x40; ++i) {
+          repl = sprintf("%c", i);
+          if ((repl == "&") || (repl == "\\")) {
+            repl = "\\" repl;
+          }
+          out = gensub(sprintf("%%%02X", i), repl, "g", out);
+          out = gensub(sprintf("%%%02x", i), repl, "g", out);
+        };
+        out = gensub(/\"/, "\\\"", "g", out);
+        print out}')
+
+      matched=$(expr match "${path}" "$MOUNT_POINT.*")
+      if [ \( "${type}" == "PUT" -o "${type}" == "DELETE" -o "${type}" == "MKCOL" -o "${type}" == "MOVE" -o "${type}" == "COPY" \) -a ${matched} != 0 ]
+      then
+        path=$(echo "$path" | cut -c ${MOUNT_POINT_LEN}- )
+        if [ "${dest}" != "-" -a "${dest}" != "" ]
+        then
+          dest=$(echo "$dest" | cut -c ${MOUNT_POINT_LEN}- )
+        fi
+
+        OP="Unknown"
+        if [ "${status}" == "201" -a "${type}" == "PUT" ]
+        then
+          OP="CREATE"
+        elif [ "${status}" == "201" -a "${type}" == "MKCOL" ]
+        then
+          OP="CREATE"
+        elif [ "${status}" == "201" -a "${type}" == "MOVE" ]
+        then
+          OP="MOVE"
+        elif [ "${status}" == "201" -a "${type}" == "COPY" ]
+        then
+          OP="CREATE"
+        elif [ "${status}" == "204" -a "${type}" == "PUT" ]
+        then
+          OP="UPDATE"
+        elif [ "${status}" == "204" -a "${type}" == "DELETE" ]
+        then
+          OP="DELETE"
+        fi
+        BODY="{}"
+        if [ "${dest}" == "-" -o "${dest}" == "" ]
+        then
+          BODY="{\"mount\":\"${MOUNT_POINT}\",\"operation\":\"$OP\",\"path\":\"${path}\"}"
+          echo "Send notification to ${ENDPOINT}: ${BODY}" > /proc/self/fd/1
+        else
+          BODY="{\"mount\":\"${MOUNT_POINT}\",\"operation\":\"$OP\",\"path\":\"${path}\",\"destination\":\"${dest}\"}"
```

Comment:
Notifications that have a Folder in $path or $dest are inconsistent in the formatting.
Please ensure consistency with the file type style of writing (e.g. all folders having a trailing slash *or* not having one, your choice)

<img width="1550" alt="inconsistent path_dest names" src="https://github.com/4ALLPORTAL/helm-charts/assets/113340892/d9da8e92-1668-4512-be59-26d7122a498a">